package.json

```diff
@@ -18,12 +18,15 @@
     "Richard Chamberlain <richard_chamberlain@uk.ibm.com> (https://github.com/rnchamberlain)"
   ],
   "scripts": {
+    "lint": "node-style",
+    "pretest": "npm run lint",
     "test": "tap test/test*.js"
   },
   "bugs": {
     "url": "https://github.com/nodejs/nodereport/issues"
   },
   "devDependencies": {
+    "node-style": "^1.1.0",
```

Comment:
@hhellyer node-style is added as a devDependency here (i.e. it won't show up if you `npm i llnode`, but it will show up if you `cd llnode && npm i`.